PlaceRouteHierFlow/toplevel.cpp

```diff
@@ -261,7 +261,10 @@ int route_top_down(PnRdatabase& DB, const PnRDB::Drc_info& drcInfo, PnRDB::bbox
 
   // 8.transform (translate and rotate) current_node into current_node coordinate
   // undo transform current_node.LL and current_node_ort
-  DB.TransformNode(current_node, current_node.LL, current_node.abs_orient, PnRDB::TransformType::Backward);
```

Comment:
@854768750 @Lastdayends Can you check if this is an okay thing to do? We are not performing this backward (on a top cell) transformation when the GDS, DB, etc. are generate. So I think we should also skip this transformation in the stored DB.

Reply:
This is fine. This reduces running this function by one time.